packages/framework/oldest-client-observer/src/oldestClientObserver.ts

```diff
@@ -104,7 +104,7 @@ export class OldestClientObserver extends TypedEventEmitter<IOldestClientObserve
             return false;
         }
 
-        assert(this.observable.clientId !== undefined, 0x1da /* "Client id should be set if connected" */);
```

Comment:
This only happened for a few asserts, I'd say revert here and then maybe do a single PR that touches them all? Also confused why only these were rewritten and not all of them.

Reply:
That's fair. The reason it affects these is because the rule is only enabled in our `recommended` config, which this package was upgraded to use due to it having minimal violations. 

I'll go ahead and revert these for now.

Reply:
I also wonder how this will interact with tagging new asserts during a release. The steps for that will require running prettier/format after policy-check so these get rewritten, right?

Reply:
I'm actually wondering if we may want to disable this rule for hex specifically (which the rule allows) to avoid this problem. Running format won't fix this, since it's an eslint policy and not a prettier one. I'll file a task to follow up on this with Tyler B. after the new year to see if he has opinions.